test/lib/commands/exec.js

```diff
@@ -254,3 +254,27 @@ t.test('npx --no-install @npmcli/npx-test', async t => {
     )
   }
 })
+
+t.test('packs from git spec', async t => {
+  const spec = 'test/test#111111aaaaaaaabbbbbbbbccccccdddddddeeeee'
+  const pkgPath = path.resolve(__dirname, '../../fixtures/git-test.tgz')
```

Comment:
I'm not the biggest fan of having this tgz file in fixtures, but if I weigh it against the effort required to build it during tests it makes sense.